src/main/java/com/rultor/agents/github/qtn/QnReferredTo.java

```diff
@@ -82,20 +84,28 @@ public QnReferredTo(final String self, final Question qtn) {
     @Override
     public Req understand(final Comment.Smart comment,
         final URI home) throws IOException {
-        final String prefix = String.format("@%s ", this.login);
+        final String prefix = String.format("@%s", this.login);
         final Req req;
-        if (comment.body().trim().startsWith(prefix)) {
-            req = this.origin.understand(comment, home);
-        } else if (comment.body().contains(prefix)) {
-            new Answer(comment).post(
-                true,
-                String.format(
-                    QnReferredTo.PHRASES.getString("QnReferredTo.mentioned"),
-                    prefix
-                )
-            );
-            Logger.info(this, "mention found in #%d", comment.issue().number());
-            req = Req.DONE;
+        final Matcher matcher = Pattern.compile(
+            String.format(".*\\b?(%s\\b).*", prefix)
```

Comment:
@mkordas  Added additional testcases and actually corrected the regex. \b is not a valid delimiter to the left e.g. ,@original-brownbear contrary to my assumption does not work :)